File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string result= "";
        for(int i =0; i<s.size(); i++){
            if(isalnum(s[i])){
                result += tolower(s[i]);

            }
        }

        int left = 0;
        int right = result.size()-1;
        while(left<right){
            if(result[left] != result[right]){
                return false;
            }
            left++;
            right--;
        }
        return true;
    }
};

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        ans = s.lower()
        ans = re.sub(r'[^a-zA-Z0-9\s]', '', ans)
        ans = ans.replace(" ", "")
        left = 0
        right = len(ans) -1

        while left < right:
            if ans[left] != ans[right]:
                return False
            left += 1
            right -= 1
        return True
    
        """
        :type s: str
        :rtype: bool
        """
        